### DISCUSSION TITLE
WG recurring meetings: 1st & 3rd Wednesdays at 19:00 GMT

### DISCUSSION THREAD

**Discussion by lloydchang** [open-gitops/project#33] [Q&A] [answered]
Do you agree with @JaimeMagiera's suggestion?

@JaimeMagiera suggests:
> Would it be better to make GitOps Working Group meetings "every first and third Wednesday"?  (or second and fourth)

**Comment by lloydchang** [comment]
From [#wg-gitops Slack 🧵](https://cloud-native.slack.com/archives/C01G9DEE88M/p1635541486026000?thread_ts=1635536574.025200)

@phillamb168 asked:
> The working group meeting says 'every other wednesday' but doesn't say starting from when - is the next one next week, or the following week?

@JaimeMagiera suggests:
> Would it be better to make it "every first and third Wednesday"? (or second and fourth)

@christianh814 and @scottrigby suggest discussing in GitHub

@lloydchang replied:
> _With 20/20 hindsight, we can laugh at ourselves 🙂 For light-hearted humor… 😆_
>
> _This actually happened in April:_
> > _Meeting About Meetings_ by GitOps Working Group
> > https://www.youtube.com/watch?v=c_GqwvP5Wqw
>
> _This is sketch comedy:_
> > _Meeting About Meetings_ by Chico Comedy
> > https://www.youtube.com/watch?v=R50tXblvMHg

**Comment by lloydchang** [comment]
@scottrigby [wrote:](https://cloud-native.slack.com/archives/C01G9DEE88M/p1636220771095200?thread_ts=1635536574.025200)
> [does someone want to put this on the agenda google doc for the next meeting?](https://cloud-native.slack.com/archives/C01G9DEE88M/p1636220771095200?thread_ts=1635536574.025200)

Done.  Asynchronously, I put this on the agenda for **Nov 10, 2021 - WG Fortnightly** via Suggesting access because I don't have Editing access at https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.pauqxhcsoepf

> Topics:
> * (Asynchronous) [Meeting About Meetings #33](https://github.com/open-gitops/project/discussions/33)
>   * [https://github.com/open-gitops/project/discussions/33 ](https://github.com/open-gitops/project/discussions/33)
>   * Do you agree with @JaimeMagiera's suggestion?

**Reply by scottrigby** [reply]
@lloydchang would you please dm me the email address you use for Google docs? Will add you as an editor ✍️

**Comment by scottrigby** [comment]
Accepted in agenda 👍

**Comment by christianh814** [comment, marked as answer]
I am +1 on this.  My instinct is to have it the 1st and 3rd Wed of every month.

This, implicitly, means that we'll have less meetings. I'm not saying it's good or bad, but it's something to consider.

**Reply by JaimeMagiera** [reply]
Agree on 1st and 3rd.

**Reply by lloydchang** [reply]
Related, I suggested changes to meeting notes headers:
> recurring ~Fortnightly~
> 1st & 3rd ~Every other~ Wednesdays at 19:00 GMT

at https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.pauqxhcsoepf

**Comment by mikecali** [comment]
I am keen to join this on a regular basis but I am based in NZ.
Will make it work thanks @lloydchang

**Comment by lloydchang** [comment]
Please review, approve, merge:
1. https://github.com/cncf/tag-app-delivery/pull/166
2. https://github.com/gitops-working-group/gitops-working-group/pull/181
3. https://github.com/open-gitops/website/pull/46

Thank you!

**Comment by lloydchang** [comment]
@chris-short asked:
> should we have a doc for the places the meeting is listed?

No.  This discussion web page is sufficient (instead of a doc), and here are 10 places that I am aware of where the meeting is listed:
1. https://www.cncf.io/calendar/
2. https://tockify.com/cncf.public.events/detail/376/1638385200000
3. https://tockify.com/cncf.public.events/detail/377/1639594800000
4. https://opengitops.dev/get-involved
5. https://github.com/cncf/tag-app-delivery/tree/main/gitops-wg#meetings
6. https://github.com/gitops-working-group/gitops-working-group#meetings
7. https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit
8. https://github.com/open-gitops/project/discussions/33
9. https://cloud-native.slack.com/archives/C01G9DEE88M/p1638120914215700?thread_ts=1638093717.215200
10. https://cloud-native.slack.com/archives/C01G9DEE88M/p1638384382231100

In the above:
• 2. and 3. tockify.com are embedded in 1. cncf.io
• 6. [github.com/gitops-working-group](https://github.com/gitops-working-group/gitops-working-group#meetings) will disappear after its content is merged into 5. [github.com/tag-app-delivery/tree/main/gitops-wg](https://github.com/cncf/tag-app-delivery/tree/main/gitops-wg#meetings)

**Comment by lloydchang** [comment]
11. @scottrigby [updated a Google Calendar](https://cloud-native.slack.com/archives/C01G9DEE88M/p1638989340258800)
